static_analysis.bash

```diff
@@ -1,11 +1,22 @@
 set -ex
 
 npx eslint \
-    'tests/e2e/**/*.ts'
+    'tests/e2e/**/*.ts' \
+    'src/composables/**/*.ts' \
+    'src/components/validated_input/**/*.vue' \
+    'src/demos/new_validated_input_demo/**/*.vue' \
+    'tests/test_components/test_new_validated_input.ts' \
+    'tests/test_components/test_new_validated_form.ts'
 
 npx prettier --check \
     './*.js' \
     './*.ts' \
-    'tests/e2e/**/*.ts'
+    'tests/e2e/**/*.ts' \
+    './src/composables/**/*.ts' \
```

Comment:
`./` not needed?

Reply:
See [a2f0d06](https://github.com/eecs-autograder/ag-website-vue/pull/534/commits/a2f0d063e478a418a34dc1b70101abf889c5c562)